### PR TITLE
update Ug from PE dry run

### LINKED ISSUE #268: [PE-D] Session Add Feature (User Guide inconsistency)
This part of the User Guide gives an example for add session.

However, on using the command in the command prompt, I receive an error because the format of date is incorrect.

I suggest that the UG be updated to reflect the command format more accurately!

-------------
Labels: `severity.High` `type.DocumentationBug`
original: souwmyaa/ped#1

### LINKED ISSUE #271: [PE-D] Unable to add data in yyyy-MM-dd format
Following the example from the UG:

`session add m/CS2103T s/14:00 e/16:00 d/19-02-2020 t/consultation n/with Alice and Bob`

Yields 

`Dates should be in yyyy-MM-dd format` error

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: JKOK005/ped#2

### LINKED ISSUE #276: [PE-D] The date for `session add` command in the user guide is not in the proper format
Referring to the first screenshot, the date format that the command accepts is in `yyyy-MM-dd` format. 

However, the user guide (refer to the second screeenshot), provides an example of adding the session with the format of `dd-MM-yyyy` format.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: mario7lorenzo/ped#15

### LINKED ISSUE #306: [PE-D] Documentation error
Example in 5.3.4 not in correct format: `done 25` should be `session done 25`

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: ZhangYiping126/ped#8

### LINKED ISSUE #309: [PE-D] Example in UG in wrong format
`session add m/CS2103T s/14:00 e/16:00 d/19-02-2020 t/consultation n/with Alice and Bob`
in section 5.3.1

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: ZhangYiping126/ped#5

### PR DESCRIPTION
**Pull request by chuayijing**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -896,7 +897,10 @@ Edits the session at index 1 to have:
 
 You can use this command to mark a session as done.
 
-The session will automatically appear as a new claim in the *Claims Vieww*.
+If the session marked done has a recurring period, a new session with the updated date
+and timing will be added to Session View.
+The session marked done will be removed from Session View and
+will automatically appear as a new claim in the *Claims View*.
```
Comment (fatin99) [inline docs/UserGuide.adoc]: maybe you can bold "Session View" here too since Claims View has been bolded

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -896,7 +897,10 @@ Edits the session at index 1 to have:
 
 You can use this command to mark a session as done.
 
-The session will automatically appear as a new claim in the *Claims Vieww*.
+If the session marked done has a recurring period, a new session with the updated date
+and timing will be added to Session View.
+The session marked done will be removed from *Session View* and
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: from the

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -827,13 +827,14 @@ Format: `session add m/MOD_CODE [s/START] [e/END] [d/DATE] [w/WEEKS] [t/SESSION_
...
 
-* `session add m/CS2103T s/14:00 e/16:00 d/19-02-2020 t/consultation n/with Alice and Bob`
+* `session add m/CS2103T s/14:00 e/16:00 d/2020-06-20 w/2 t/consultation n/with Alice and Bob`
 +
 Adds a new session with:
 
 ** `START_TIME` at 14:00
 ** `END_TIME` at 16:00
-** `DATE` on 19-02-2020
+** `DATE` on 2020-06-20
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Maybe this could be in `12hr` format and `dd MMM yyyy` format?

File: docs/UserGuide.adoc (hunk 2 above)
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Maybe the two could be swapped? The session marked done moves to claims, then if recurring repeat.

**Review by PotatoCombat — COMMENTED** [review]
Maybe can change these?

**Review by PotatoCombat — APPROVED** [review]
LGTM!

**Pull request merged**